src/Components/HighChartsWrapper/HighChartsWrapper.types.ts

```diff
@@ -0,0 +1,93 @@
+import { IStyle, IStyleFunctionOrObject, ITheme } from '@fluentui/react';
+import { OptionsLayoutValue } from 'highcharts';
+import { TimeSeriesData } from '../../Models/Constants';
```

Comment:
it is in Constants/Types.ts file

Reply:
oh it's the barrel import. Can we import from the file directly so that this file does not include all of the files in that folder? It helps when the browser goes to parse the file.